lib/process-services-cloud/src/lib/form/services/process-cloud-content.service.ts

```diff
@@ -45,6 +45,18 @@ export class ProcessCloudContentService {
       );
   }
 
+  getRawContentNode(nodeId: string, contentHost: string): Observable<any> {
+    const changedConfig = this.apiService.lastConfig;
+    changedConfig.provider = 'ALL';
```

Comment:
does it modify the config forever or just for this call?

Reply:
it is forever but for the cloud should not be a problem. What is your concern?

Reply:
The concern is that the service modifies globally the provider for the entire application. That might not be an expected behavior when ECM is not configured but it will require the login?

Reply:
As per design now the cloud is working with keyclock as auth provider, so at design time you will decide which content you want to have attached to your application. This means that the idea of changing the app config to try to connect to a different content loose it's validity into the cloud side. What we do here is (even though it's not the best and beautyfull way) to basically connect to the content the user configured at design time to fetch the information needed